pkg/kubefed2/join.go

```diff
@@ -489,19 +500,22 @@ func createRBACSecret(hostClusterClientset, joiningClusterClientset kubeclient.I
 		glog.V(2).Infof("Created role and binding for service account: %s in joining cluster: %s",
 			saName, joiningClusterName)
 
-		glog.V(2).Infof("Creating health check cluster role and binding for service account: %s in joining cluster: %s", saName, joiningClusterName)
-
-		err = createHealthCheckClusterRoleAndBinding(joiningClusterClientset, saName, namespace, joiningClusterName,
-			dryRun, errorOnExisting)
-		if err != nil {
-			glog.V(2).Infof("Error creating health check cluster role and binding for service account: %s in joining cluster: %s due to: %v",
-				saName, joiningClusterName, err)
-			return nil, err
-		}
+		if hostClusterName == joiningClusterName {
+			glog.V(2).Infof("Skipping creating health check cluster role and binding for host cluster: %s", hostClusterName)
```

Comment:
Could you help to explain below questions to me:
 - What is the reason to skip this on host cluster? 
 - This role binding is used for helth check in controller. What is the relationship with the namespace creation case?

Reply:
As discussed, I've removed this part of the PR.